pom.xml

```diff
@@ -95,44 +95,55 @@
       <dependency>
         <groupId>org.commonjava.maven.ext</groupId>
         <artifactId>pom-manipulation-common</artifactId>
-        <version>3.9-SNAPSHOT</version>
+        <version>${project.version}</version>
       </dependency>
       <dependency>
```

Comment:
Why this change?

Reply:
This was to make sure that these dependencies automatically keep the version in sync with the project version since these dependencies should always come from the project build itself and not an external maven repository.

Reply:
The release plugin handles that - see http://central.maven.org/maven2/org/commonjava/maven/ext/pom-manipulation-parent/3.8.1/pom-manipulation-parent-3.8.1.pom for example. Also using $project.version is really not advised (which is why PME inlines it when it encounters it). Please revert this.

Reply:
> The release plugin handles that - see http://central.maven.org/maven2/org/commonjava/maven/ext/pom-manipulation-parent/3.8.1/pom-manipulation-parent-3.8.1.pom for example. Also using $project.version is really not advised (which is why PME inlines it when it encounters it). Please revert this.

This doesn't make a lot of sense to me as there are already five or more version properties defined in the pom, presumably for the same reason, to avoid errors and repeatedly hardcoding the same value over and over.

Reply:
Those aren't using project.version and, as I said, the release plugin handles transforming this. Its done on purpose ;-)